admin/choose_project.php

```diff
@@ -26,5 +33,17 @@
         <input type="submit" value="Choose project">
     </form>
 
+    <form method="POST" action="delete_project.php" onsubmit="return confirmationBox();">
+        <label>Select a project to <strong>DELETE</strong></label>
+        <select name="id">
```

Comment:
this select is identical to the one above, you can put them into a function and then call in both places